backend/internal/data/migrations/postgres/20251128120101_drop_item_templates.sql

```diff
@@ -0,0 +1,3 @@
+-- +goose Up
```

Comment:
So the only thing here is that the SQLite and Postgres have 3 migrations each for this change.

I know that this may have been needed in a test case/playing with things. However, because this code hasn't yet been published, to production, one migration for each type should work (the final one "recreate" one.

Simply deleting the extra ones, and renaming the "recreate" one should suffice in that area.

Because we can't ever delete migrations (at least not without breaking the migration path from Hay-Kots version to our latest versions) keeping the migrations as clean as possible before they hit the main branch helps maintain things.